Add polynomial em properties (#2088)
chaos-polymtl/lethe: release_notes/current/2026_08_11_Marquis.md

## [Master] - 2026/08/11

### Added

> - MAJOR The time-harmonic Maxwell solver now has temperature dependent physical properties using polynomial model. This enables the retrocoupling of heat transfer when performing microwave heating simulation since they now affect the electromagnetic properties. New tests for the different electromagnetic properties have been added to Lethe test suite. [#2088](https://github.com/chaos-polymtl/lethe/pull/2088)

Reviewer: ```suggestion
- MAJOR The time-harmonic Maxwell solver now has temperature dependent physical properties using polynomial model. This enables the retrocoupling of heat transfer when performing microwave heating simulation since they now affect the electromagnetic properties. New tests for the different electromagnetic properties have been added to the Lethe test suite. [#2088](https://github.com/chaos-polymtl/lethe/pull/2088)
```

> - MAJOR The time-harmonic Maxwell solver now has temperature dependent physical properties using polynomial model. This enables the retrocoupling of heat transfer when performing microwave heating simulation since they now affect the electromagnetic properties. New tests for the different electromagnetic properties have been added to Lethe test suite. [#2088](https://github.com/chaos-polymtl/lethe/pull/2088)

Reviewer 2: ```suggestion
- MAJOR The time-harmonic Maxwell solver now has temperature-dependent physical properties using a polynomial model. This enables the retrocoupling of heat transfer when performing microwave heating simulation since they now affect the electromagnetic properties. New tests for the different electromagnetic properties have been added to Lethe test suite. [#2088](https://github.com/chaos-polymtl/lethe/pull/2088)
```

> - MAJOR The time-harmonic Maxwell solver now has temperature dependent physical properties using polynomial model. This enables the retrocoupling of heat transfer when performing microwave heating simulation since they now affect the electromagnetic properties. New tests for the different electromagnetic properties have been added to the Lethe test suite. [#2088](https://github.com/chaos-polymtl/lethe/pull/2088)

Reviewer 2: ```suggestion
- MAJOR The time-harmonic Maxwell solver now has temperature-dependent physical properties using a polynomial model. This enables the retrocoupling of heat transfer when performing microwave heating simulation since they now affect the electromagnetic properties. New tests for the different electromagnetic properties have been added to the Lethe test suite. [#2088](https://github.com/chaos-polymtl/lethe/pull/2088)
```

> - MINOR The coupling strategy "treshold" is now implemented. The electromagnetic physics now records the temperature field solution the last time the time-harmonic solver was use to solve the electromagnetic field and use it to measure the electromagnetic properties relative change since the last solve. It then compare this change to the user define threshold to flag a neew solve or not. [#2088](https://github.com/chaos-polymtl/lethe/pull/2088)

Reviewer: ```suggestion
- MINOR The coupling strategy "treshold" is now implemented. The electromagnetic physics now records the temperature field solution at the last time when the time-harmonic solver was use to solve the electromagnetic field and uses it to measure the electromagnetic properties relative change since the last solve. It then compares this change to the user define threshold to flag a new solve or not. [#2088](https://github.com/chaos-polymtl/lethe/pull/2088)
```

> - MINOR The coupling strategy "treshold" is now implemented. The electromagnetic physics now records the temperature field solution the last time the time-harmonic solver was use to solve the electromagnetic field and use it to measure the electromagnetic properties relative change since the last solve. It then compare this change to the user define threshold to flag a neew solve or not. [#2088](https://github.com/chaos-polymtl/lethe/pull/2088)

Reviewer 2: ```suggestion
- MINOR The coupling strategy "threshold" is now implemented. The electromagnetic physics now records the temperature field solution the last time the time-harmonic solver was used to solve the electromagnetic field and use it to measure the electromagnetic properties relative change since the last solve. It then compares this change to the user-define threshold to flag a new solve or not. [#2088](https://github.com/chaos-polymtl/lethe/pull/2088)
```

> - MINOR The coupling strategy "treshold" is now implemented. The electromagnetic physics now records the temperature field solution at the last time when the time-harmonic solver was use to solve the electromagnetic field and uses it to measure the electromagnetic properties relative change since the last solve. It then compares this change to the user define threshold to flag a new solve or not. [#2088](https://github.com/chaos-polymtl/lethe/pull/2088)

Reviewer 2: ```suggestion
- MINOR The coupling strategy "threshold" is now implemented. The electromagnetic physics now records the temperature field solution at the last time when the time-harmonic solver was used to solve the electromagnetic field and uses it to measure the electromagnetic properties relative change since the last solve. It then compares this change to the user-define threshold to flag a new solve or not. [#2088](https://github.com/chaos-polymtl/lethe/pull/2088)
```

> - MINOR The coupling strategy "threshold" is now implemented. The electromagnetic physics now records the temperature field solution at the last time when the time-harmonic solver was used to solve the electromagnetic field and uses it to measure the electromagnetic properties relative change since the last solve. It then compares this change to the user-define threshold to flag a new solve or not. [#2088](https://github.com/chaos-polymtl/lethe/pull/2088)

Reviewer: ```suggestion
- MINOR The coupling strategy "threshold" is now implemented. The electromagnetic physics now records the temperature field solution at the last time when the time-harmonic solver was used to solve the electromagnetic field and uses it to measure the electromagnetic properties relative change since the last solve. It then compares this change to the user-defined threshold to flag a new solve or not. [#2088](https://github.com/chaos-polymtl/lethe/pull/2088)
```
